### PR TITLE
PS-8217 Document secure-log-path system variable (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M  source/index.rst
A  source/security/system-variables.rst

### CODE REVIEW

File: doc/source/security/system-variables.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Variable Name
+     - Description
+   * - Command-line
+     - --secure-log-path
+   * - Dynamic
+     - Yes
```
Comment (dutow) [inline doc/source/security/system-variables.rst]: It's not dynamic.

File: doc/source/security/system-variables.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   * - Scope
+     - Global
+   * - Data type
+     - String
+   * - Default
+     - empty string
+     
+This variable restricts the dynamic log file locations. The variable is read-only and must be set up in a configuration file or the command line. 
+
+The accepted value is the directory name as a string. The default value is an empty string. When the value is an empty string, the variable only adds a warning to the error log and does nothing. If the value contains a directory name, then the slow query log and the general log must be located in that directory. An attempt to move either of these files results in an error.
```
Comment (dutow) [inline doc/source/security/system-variables.rst]: "An attempt to move either of these files" - outside of the specified directory.

**Review by dutow — CHANGES_REQUESTED** [review]

File: doc/source/security/system-variables.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/security/system-variables.rst]: Fixed

File: doc/source/security/system-variables.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/security/system-variables.rst]: Added.

**Review by dutow — APPROVED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**